de-lib/src/main/java/org/iplantc/de/tools/client/presenter/ToolSharingPresenterImpl.java

```diff
@@ -192,16 +192,13 @@ public void processRequest() {
         ToolUnSharingRequestList unshareRequest = buildUnSharingRequest();
         if (request != null) {
             callSharingService(request);
+            IplantAnnouncer.getInstance().schedule("Sharing request submitted");
```

Comment:
This and a few lines below can have the `IplantAnnouncer` injected and the messages should probably go into an appearance